docker-compose.yaml

```diff
@@ -19,16 +19,16 @@ services:
       args:
         ALPINE_VERSION: "3.16"
         PHP_VERSION: "8.0"
-        PHP_XDEBUG_VERSION: "3.1.2"
+        PHP_XDEBUG_VERSION: "3.2.2"
```

Comment:
question: can we bump it also for PHP 7.4 ?

Reply:
Yes, but to 3.1.6 (done in [bb6a401](https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/pull/7300/commits/bb6a401d358376c31c79c86ad4a2e41c41dd4d65)), because XDebug [3.2.0](https://pecl.php.net/package-info.php?package=xdebug&version=3.2.0) dropped support for 7.4 🙂.